lib/http-revocations.js

```diff
@@ -21,11 +23,10 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   app.options(routes.revocations, cors());
   app.post(
     routes.revocations,
+    validate({bodySchema: zcap}),
     // CORs is safe because revocation uses HTTP signatures + capabilities,
     // not cookies
     cors(),
```

Comment:
`cors()` middleware needs to come first otherwise a failed validation error won't be received by a browser making a cross-origin call, as the CORS headers won't be sent.